contracts/bridge_test.go

```diff
@@ -318,16 +330,12 @@ func TestBridgePublicVariables(t *testing.T) {
 	chargeAmount := big.NewInt(10000000)
 	bridgeAccount.Value = chargeAmount
 	bridgeAddress, tx, b, err := bridge.DeployBridge(bridgeAccount, backend, false)
-	if err != nil {
-		t.Fatalf("fail to DeployBridge %v", err)
-	}
+	assert.Equal(t, nil, err)
```

Comment:
what about using assert.NoError(t, err)?

Reply:
I will make a PR for all code. :) Thanks.

Reply:
Anyway, I reflect to this code. kaka